### PR TITLE
feat: account linking docs

### PR DESCRIPTION
**Pull request by hperl**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/social-signin/95_account-linking.mdx [hunk 1]
```diff
@@ -46,15 +46,27 @@ To try out account un-linking, use the Ory Account Experience. Follow these step
...
-accounts.
-
-:::warning
+:::info
 
-Since it constitutes a security threat, automatic account linking is not available in Ory.
+Automatic account linking was previously not available in Ory. It has been recently added and is now enabled on the Ory Network as
+well as on the open-source master branch.
 
 :::
```
Comment (jonas-jonas) [inline docs/kratos/social-signin/95_account-linking.mdx:54]: I think we can just remove this disclaimer

Comment (hperl) [reply docs/kratos/social-signin/95_account-linking.mdx:54]: Fine by me. I wanted to leave it in because it was not possible for a long time.

Comment (vinckr) [reply docs/kratos/social-signin/95_account-linking.mdx]: I agree we dont need the disclaimer here, we can point this out in the changelog/newsletter/community/etc.

**Review by vinckr — DISMISSED** [review]
lgtm =)

**Review by jonas-jonas — DISMISSED** [review]

**Review by jonas-jonas — APPROVED** [review]

**Pull request merged**